client/src/app/create/create.component.ts

```diff
@@ -3,24 +3,25 @@ import {EventEmitter} from "@angular/core";
 import {FormGroup, FormBuilder, Validators} from "@angular/forms";
 import {Word} from '../models/word';
 import {WordListService} from "../services/wordList.service";
+import {Observable} from "rxjs";
 
 @Component({
   selector: 'app-create',
   templateUrl: './create.component.html',
   styleUrls: ['./create.component.scss']
 })
 
+
+
 export class CreateComponent implements OnInit {
+  wordList: Observable<Word[]>;
   newWord: FormGroup;
   activeList: boolean = false;
-  wordList: Word[];
 
   constructor(private fb: FormBuilder,
               private wordListService: WordListService) {
     this.createForm();
-    this.wordListService.wordList$.subscribe( results =>{
-      this.wordList = results;
```

Comment:
reverse bez pipe
`this.wordListService.wordList$.subscribe( results =>{ this.wordList = results.slice().reverse(); });`